docs/src/main/paradox/discovery/kubernetes.md

```diff
@@ -72,9 +69,9 @@ spec:
         - name: remoting
           containerPort: 2552
           protocol: TCP
-        # akka-management bootstrap
-        # This name must match the name defined in
-        # akka.discovery.kubernetes-api.pod-port-name configuration
+        # When
```

Comment:
Why is this when and not a mandatory property? Won't we get back all the ports if None is passed in, does that ever make sense?

Reply:
In the non-discovery use case, indeed we get all ports back if None is passed in, which might make sense in situations where a service opens a single port, so the service name uniquely identifies the service and no port name is needed to discover it.

In the discovery case, we either 1) look for the `management` port name or 2) look for the port akka-management is configured to listen on. I think I prefer '1', but also prototyped '2' in #503 to see how that turns out.

Reply:
:+1: